### PR TITLE
Alpha5 Release Notes

### PR DESCRIPTION
**Pull request by akeller**
## Description

Initia pull C&P from [Product Hub](https://github.com/orgs/camunda/projects/9/views/31?filterQuery=label%3A%22target%3A8.6-alpha5%22%2C%22version%3A8.6-alpha5%22%2C%22potential%3A8.6-alpha5%22+label%3A%22release-notes-needed%22).

Related to https://github.com/camunda/engineering-leadership-team/issues/203

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
Looking for feedback on badges - https://camunda.slack.com/archives/C026U8GBNSW/p1724784231042549

**Comment by mesellings** [comment]
@akeller I wonder whether we should group features and items under headings, e.g. Help Center, Modeler, Zeebe etc as well as with badges, as currently I still have to scroll the page to see every item, whereas if they are grouped, I can quickly scan the areas that are important for me? I could do this for you if you like, to see if it is a viable idea?

**Comment by akeller** [comment]
> @akeller I wonder whether we should group features and items under headings, e.g. Help Center, Modeler, Zeebe etc as well as with badges, as currently I still have to scroll the page to see every item, whereas if they are grouped, I can quickly scan the areas that are important for me? I could do this for you if you like, to see if it is a viable idea?

Let's follow up on this in a separate issue/PR @mesellings. Since there isn't a 1:1 mapping for features to components, we may need to iterate internally on how to group these.

**Comment by akeller** [comment]
Can I also have reviewers look at the badges for each epic in addition to a grammar/spelling review? This won't exactly match the component labels on the epic, but it should cover the primary component AND include Self-Managed or SaaS if obviously relevant.

You can also verify the items included are corrected based on [this Product Hub view](https://github.com/orgs/camunda/projects/9/views/31?filterQuery=label%3A%22target%3A8.6-alpha5%22%2C%22version%3A8.6-alpha5%22%2C%22potential%3A8.6-alpha5%22+label%3A%22release-notes-needed%22++).

We have no release blurb for one topic, but more may trickle in. Moving this out of draft, but adding a `hold` label.

**Comment by mesellings** [comment]
@akeller I'll review this and also add the Auto-pause entry (confirmed by Eric) 👍

**Comment by mesellings** [comment]
@akeller I also left the Tech-writers reviewer tag on there, wasn't sure if you wanted multiple reviewers?

**Comment by mesellings** [comment]
@akeller I've gone through the alpha 5 release notes, and they line up correctly with the issues in the product hub, other than the auto-pause feature (which is incorrectly labeled, but confirmed by Eric as being needed) and the content required for the core styling item.

I've made edits as follows:
* Added an intro blurb to the page
* Edited headings to front-load the relevant content area as much as possible, for easier scanning
* Edited/restructured content to try and reduce cognitive load when faced with a wall of text/paragraphs etc
* Badges all look good/relevant to me

**Comment by mesellings** [comment]
@akeller I added a fix to the custom.css in this PR as the current implementation for hiding the badges in the TOC used `visibility: hidden` which still kept the badge <span> elements in the TOC, causing some extra spacing to appear:

image

I've changed this to `display:none` so they are completely removed from the page layout which fixes this issue.

**Review by mesellings — APPROVED** [review]
Lgtm - approved 👍

File: docs/reference/release-notes/860.md [hunk 1]
```diff
@@ -11,19 +11,136 @@ keywords:
...
+- The interface language is automatically determined by the user device's language settings.
+- The interface defaults to English if the device's language is not supported.
+
+### Message correlation with result <span class="badge badge--long" title="This feature affects Zeebe">Zeebe</span>
+
+<!-- https://github.com/camunda/product-hub/issues/1818 -->
+
+Real-time feedback is added for message correlation for messages with `ttl=o`, enabling external systems to immediately determine the success or failure of message correlation. This enhancement allows external systems to take prompt and appropriate actions based on the correlation result, improving overall efficiency and reducing response times.
+
+### Public Marketplace Blueprint support for HTO & DMN <span class="badge badge--long" title="This feature affects Web Modeler">Web Modeler</span> <span class="badge badge--long" title="This feature affects Marketplace">Marketplace</span>
```
Comment (christinaausley) [inline docs/reference/release-notes/860.md:85]: @YanaSegal have we decided to capitalize Blueprints for a particular reason here or is there a related thread? IMO this should be lowercase, but open to opinions. CC @camunda/tech-writers

Comment (YanaSegal) [reply docs/reference/release-notes/860.md:85]: @christinaausley 
I think there is no reason for capitalizing blueprint - I can't recall any decision to capitalize it.

**Pull request merged**